dm140.c

```diff
@@ -0,0 +1,241 @@
+/*
+ *  dm1400 vfd driver (c)2007 Henrik Larsson
+ */
```

Comment:
Um, what's the license of this code? And I think it is lcdproc policy to only merge code with the actual authors consent...